test/user-profile/helpers.unit.spec.js

```diff
@@ -77,7 +77,7 @@ describe('profile helpers:', () => {
       expect(isSIPEnabledForm('missingInfoForm')).to.be.false;
     });
     it('should throw an error if a form is not included the list of sipEnabledForms', () => {
-      expect(() => isSIPEnabledForm({ form: '22-5490' })).to.throw('Could not find form');
+      expect(() => isSIPEnabledForm({ form: '40-10007' })).to.throw('Could not find form');
```

Comment:
Suppose we should probably just make a random form id. Is that what this does?

Reply:
The test above is for random ids, this one is for forms that are in our list of forms, but don't have SiP on yet. I think this test will go away once pre-needs adds SiP.